packages/vaadin-avatar/src/vaadin-avatar-icons.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+const $_documentContainer = document.createElement('template');
```

Comment:
We only need to re-export `vaadin-avatar` and `vaadin-avatar-group` so this file can be removed.